src/testRunner.js

```diff
@@ -0,0 +1,65 @@
+const createTestCafe = require("testcafe");
+
+class TestRunner {
+
+  constructor(createTestCafe, browsers, testSources = ["tests/fixtures"]) {
+    this.createTestCafe = createTestCafe;
+    this.browsers = browsers;
+    this.testSources = testSources;
+  }
+
+  async runTests() {
+    let testcafe; // Outer scope due to use in `finally` block
+    try {
+      testcafe = await createTestCafe("localhost", 1337, 1338);
+      const runner = testcafe.createRunner();
+      const callingScriptPath = require('path').dirname(require.main.filename);
```

Comment:
Users might want the ability to change this dynamically, for instance if they needed to output files to a certain directory for use in a CI.

Possibly not urgent for this iteration?

Reply:
This also might be the case for this params on ln 14. `createTestCafe("localhost", 1337, 1338);`. Might not want to go overboard with the config options either I guess.